primitives/src/v5/mod.rs

```diff
@@ -1165,6 +1165,8 @@ pub struct AbridgedHostConfiguration {
 	pub validation_upgrade_cooldown: BlockNumber,
 	/// The delay, in blocks, before a validation upgrade is applied.
 	pub validation_upgrade_delay: BlockNumber,
+	/// Asynchronous backing parameters.
+	pub async_backing_params: super::vstaging::AsyncBackingParams,
```

Comment:
It seems a bit weird that v5 types depend on `vstaging` types. Also by making this change, shouldn't we bump a version? And stabilize this `vstaging` type?

Reply:
I think we should bump it along with staging runtime API becoming available. The change from this PR is backwards-compatible -- old `AbridgedHostConfig` can be decoded from the storage value as well.